content/v2.0/organizations/members/add-member.md

```diff
@@ -7,25 +7,36 @@ menu:
     name: Add a member
     parent: Manage members
 weight: 201
-draft: true
 ---
 
-Use the InfluxDB user interface (UI) or the `influx` command line interface (CLI)
-to add a member to an organization.
+Use the `influx` command line interface (CLI) to add a member to an organization.
 
-## Add a member to an organization in the InfluxDB UI
+{{% cloud-msg %}}
+Adding members is currently unavailable in {{< cloud-name >}}.
+{{% /cloud-msg %}}
+
+<!-- ## Add a member to an organization in the InfluxDB UI
 
 1. Click the **Settings** tab in the navigation bar.
 
     {{< nav-icon "settings" >}}
 
 2. Click on the name of an organization, then select the **Members** tab.
 
-_Complete content coming soon_
+_Complete content coming soon_ -->
 
 ## Add a member to an organization using the influx CLI
 
-Use the [`influx org members add` command](/v2.0/reference/cli/influx/org/members/add)
-to add a user to an organization. Adding a user requires the following:
+1. Get a list of users and their IDs by running the following:
+
+```sh
+influx user find
+```
+
+2. Add the desired user to an organization with this command:
```

Comment:
- Add the desired user **as a member of** an organization...

I think we need to be really clear that there's a distinction between a user and a member.